### ISSUE TITLE
Consider removing or making vue-virtual-scroller optional

### ISSUE THREAD

**Issue by serebrov** [serebrov/emoji-mart-vue#52]
While vue-virtual-scroller helps getting some additional performance, it is not critical and component worked fast enough without it.
The disadvantage of the virtual scroller is that it adds complex HTML structure which is harder to customize (for example, it is complex to style the scroll bar).

**Comment by briancaffey** [comment]
I have come across this issue in trying to customize the height of the scroller. I'm finding strange behavior on mobile and sometimes on desktop browsers the categories are overlapping. @serebrov thanks

**Comment by apasov** [comment]
@serebrov I support your idea. I described my situation in #57

**Comment by serebrov** [comment]
Related PR that adds virtual scroller - https://github.com/serebrov/emoji-mart-vue/pull/20

**Comment by serebrov** [comment]
@briancaffey @apasov I've just pushed version `6.1.0` to npm with new `StaticPicker` component - it is the same as `Picker`, but doesn't use the vue-virtual-scroller.

Updated [demo app](https://serebrov.github.io/emoji-mart-vue/), the "Custom Search And Preview Templates Example" example uses `StaticPicker`.

Also added related information to [README](https://github.com/serebrov/emoji-mart-vue#staticpicker).
